packages/renderer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@motion-canvas/renderer",
+  "version": "3.14.1",
+  "description": "A headless renderer Motion Canvas",
```

Comment:
I think we should change this to `A headless renderer for Revideo` or something like that.

Reply:
We still have other packages refering to motion canvas, I would change the naming in one go in a seperate PR. But I fixed the grammatical error